config/locales/en.yml

```diff
@@ -41,7 +41,7 @@ en:
       nav:
         about: "How this works"
         privacy: "Privacy Policy"
-        github: "Github"
+        github: "GitHub"
```

Comment:
I've updated this along with some additional updates to the `en.yml` file. 
I'll be closing this PR.